plugins/events/listBuilds.js

```diff
@@ -24,14 +24,23 @@ module.exports = () => ({
         handler: async (request, h) => {
             const { eventFactory } = request.server.app;
             const event = await eventFactory.get(request.params.id);
+            const { steps } = request.query;
 
             if (!event) {
                 throw boom.notFound('Event does not exist');
             }
 
-            const buildsModel = await event.getBuilds();
+            const buildsModel = await event.getBuilds({
+                readOnly: true
```

Comment:
do we need to hit `readOnly` DB instance ? shouldn't build data be always shown up to date ?